Resources/Prototypes/ADT/Catalog/Fills/Lockers/cargo.yml

```
@@ -7,7 +7,7 @@
     - id: RadioHandheld
     - id: WeaponProtoKineticAccelerator
     - id: FireExtinguisher
-    - id: SeismicCharge
+    # - id: SeismicCharge временно в ауте
```

```suggestion
    - id: SeismicCharge временно в ауте
```

Why? Утили на обломках её юзают, а вот в шахтёрмаге да, убрать стоит